### PR TITLE
Fixes #205 - Automatically remove IPv6 listen directive

### LINKED ISSUE #205: New Zammad NGINX config (#3371) breaks zammad-docker-compose: nginx: [emerg] a duplicate listen 0.0.0.0:8080
# Infos

* Docker version: Docker version 18.09.5, build e8ff056
* Docker-compose version: docker-compose version 1.21.0, build unknown
* Operating system (Docker host): Debian GNU/Linux 10 (buster)

# Expected behavior

* Upgrading/installing a `zammad-docker-compose` setup should not cause the following NGINX error:

```
zammad-nginx_1          | nginx: [emerg] a duplicate listen 0.0.0.0:8080 in /etc/nginx/sites-enabled/default:15                                                                                        
```

# Actual behavior

* Upgrading/installing a `zammad-docker-compose` setup breaks NGINX.

# Steps to reproduce the behavior

* Upgrade/install `zammad-docker-compose`.

### PR DESCRIPTION
**Pull request by MrGeneration**
This pull request addresses issue #205 which was introduced due to issue #3371 of the main zammad repo.

It removes the IPv6 listen directive *before* replacing listen directives which currently causes duplicate listen directives and thus results in crashing nginx containers.

### CODE REVIEW

File: containers/zammad/docker-entrypoint.sh [hunk 1]
```diff
@@ -121,7 +121,8 @@ if [ "$1" = 'zammad-nginx' ]; then
   check_zammad_ready
 
   # configure nginx
-  sed -e "s#listen .*#listen ${NGINX_PORT};#g" \
+  sed -e "/\[..\]:80/d" \
+      -e "s#listen .*#listen ${NGINX_PORT};#g" \
```
Comment (monotek) [inline containers/zammad/docker-entrypoint.sh:125]: The new regex is superfluous. We can do this in the existing line, using the same ipv6 config in the container as well:

```
s#\(listen\)\(.*\)80#\1\2${NGINX_PORT}#g"
```

**Review by monotek — CHANGES_REQUESTED** [review]
```
s#\(listen\)\(.*\)80#\1\28080#g"
```

**Review by monotek — APPROVED** [review]

**Review by monotek — APPROVED** [review]

**Comment by rvalle** [comment]
@monotek I just installed Zammad for first time. 

Can you release this patch to the docker image? 

I think it the currently published image is not functional without this fix.

**Comment by rvalle** [comment]
I saw the ```sed```  workaround now...

**Comment by monotek** [comment]
The image is build as soon as new commits arrive in the stable branch of https://github.com/zammad/zammad

**Pull request merged**